SECURITY.md

```diff
@@ -0,0 +1,42 @@
+# GRASS GIS Security Policy
+
+## Reporting a Vulnerability
+At GRASS GIS, we take security vulnerabilities seriously. We appreciate your efforts in responsibly disclosing any issues you may find. To report a security vulnerability, please follow these steps:
+1. **Privately disclose the issue** by emailing our [Security Team](vpetras@ncsu.edu) or by submitting a Security Advisory through [GitHub Security](https://github.com/OSGeo/grass/security). Please do not create publicly viewable issues for security vulnerabilities.
+2. **Provide detailed information** regarding the vulnerability, including:
+    - Description of the vulnerability
+    - Steps to reproduce
+    - Versions affected
+    - Any mitigating factors
+3. **Our Response**:
+    - Our security team will acknowledge receiving your report within 48 hours.
+    - We will work to validate and reproduce the issue.
+    - Once confirmed, we will work on a fix and release schedule.
+4. **Public Disclosure**:
+    - We aim to release patches for vulnerabilities as soon as possible.
+    - We will coordinate with you regarding public disclosure, ensuring a reasonable timeline for users to update before the details are made public.
+
+
+## Supported Versions
+We prioritize the most recent stable release of GRASS GIS for security patches. However, we understand that users may still be on older versions. We will make reasonable efforts to provide patches for security vulnerabilities in older releases on a case-by-case basis, depending on the severity and feasibility.
+
+| Version | Supported |
+| ------- | --------- |
+| 8.x.x   | ✅        |
+| 7.x.x   | ✅        |
+| < 7.0.0 | 🔴        |
```

Comment:
This should link our release procedure or release guidelines. I think the best document at this point is [Trac wiki page Release Schedule](https://trac.osgeo.org/grass/wiki/Release/Schedule). Replace the table by a sentence referring people to that page. Something along the lines of "Refer to [Release Schedule](https://trac.osgeo.org/grass/wiki/Release/Schedule) for details about which versions are supported."